interfaces/oauth_client/v0/schemas/provider.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
Rework these schemas to be at the root level, ie. contain a property for the application databag, and one property for the unit databag. See https://github.com/canonical/charm-relation-interfaces/blob/main/interfaces/prometheus_scrape/v0/schemas/provider.json for an example. This will eventually need to be done for all schemas.